clients/java/src/test/resources/form/test-form-1.form

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
💭 We do not really need to define a proper `.form` file here. Tests should pass with referencing an empty file since we mock the response. At the same time, I think, we should still have a proper form format to make it clear how a form should look like. 👍

Reply:
Yeah, same idea 👍